File: cpp/bc1159.cpp
#include <iostream>

using namespace std;

int main()
{
	int val, som, count;
	
	for(;;)
	{
		som = 0;
		
		cin >> val;
		
		if(val == 0) break;
		else
		{
			count = 0;
			
			while(true)
			{
				if(val % 2 == 0)
				{
					som += val;
					count++;
				}
				if(count == 5) break;
				else val++;
			}
			cout << som << endl;
		}
	}
}

File: python/bc1159.py
while True:
    soma = 0

    val = int(input())

    if val == 0: break
    else:
        count = 0

        while True:
            if val % 2 == 0:
                soma += val
                count += 1
            if count == 5: break
            else:
                val += 1

    print(soma)